docs/admin_guides/docker_admin.md

```diff
@@ -100,23 +82,4 @@ To restore a backup to a new dockerized instance of SeAT, run:
 zcat seat_backup.sql.gz | docker-compose exec -T mariadb sh -c 'exec mysql "$MYSQL_DATABASE" -u"$MYSQL_USER" -p"$MYSQL_PASSWORD"'
 ```
 
-## Performing Updates
```

Comment:
Reason for removing update information?

Reply:
No removed, refactored.
Same logic than admin login is applied here - content has been grouped and relocated here https://github.com/warlof/eveseat-docs/blob/4.0.x/docs/upgrading/general.md